apps/mull-ui/src/app/pages/direct-message/direct-message.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import { ChatHeader, CustomTextInput } from '../../components';
+import ContactRow from '../../components/contact-row/contact-row';
+import './direct-message.scss';
+
+/* eslint-disable-next-line */
+export interface DirectMessageProps {}
+
+// TODO: Query that returns an array of user profiles related to user
+const userData = [
+  {
+    id: 1,
+    name: 'Bob Marley',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+  {
+    id: 2,
+    name: 'Shawn Mendes',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+  {
+    id: 3,
+    name: 'Justin Bieber',
+    description: 'Singer',
+    picture:
+      'https://blog.photofeeler.com/wp-content/uploads/2017/04/are-bumble-profiles-fake-how-many.jpeg',
+  },
+];
+
+const DirectMessagePage = () => {
+  const [searchValue, setSearchValue] = useState('');
+
+  const handleChange = (event) => {
+    setSearchValue(event.target.value);
+  };
+
+  return (
+    <div className="direct-messages-container">
+      <ChatHeader eventTitle="Direct Messages" />
+      <div className="page-container">
+        <CustomTextInput
+          title=""
+          fieldName="searchValue"
+          value={searchValue}
+          onChange={handleChange}
+          hasErrors={null}
+          errorMessage={null}
+          placeholder={'Search'}
+        />
+        {userData
+          .filter(
+            (user) =>
+              user.name.toLowerCase().includes(searchValue.toLowerCase()) || searchValue === ''
+          )
+          .map((user) => {
+            return (
+              <ContactRow
+                userId={user.id}
+                userName={user.name}
+                userDescription={user.description}
+                userPicture={user.picture}
+              />
+            );
+          })}
```

Comment:
To avoid looping through it using a filter then mapping it, we can just run an if statement in the map function.
https://stackoverflow.com/questions/57326905/array-filter-and-map-or-map-with-a-if-condition
```suggestion
          .map((user) => {
            if (user.name.toLowerCase().includes(searchValue.toLowerCase()) || searchValue === '')
            return (
              <ContactRow
                userId={user.id}
                userName={user.name}
                userDescription={user.description}
                userPicture={user.picture}
              />
            );
          })}
```

Reply:
Done!